### DISCUSSION TITLE
Allow finishing review from the "Conversation" tab

### DISCUSSION THREAD

**Discussion by xmo-odoo** [community/community#11227] [Pull Requests]
The "files changed" tab can be excruciatingly slow for large prs (tens of thounsands of lines changed). It also gets very long, which makes finding and reviewing comments performed on the diff difficult.

The "conversations" tab provides a nice grouping of the pending review comments, but actually landing the review requires going back to the "Files Changed", waiting for it to load (again) then having to write the review without being easily able to refer to the review comments if needed.

The text box even taunts you by telling you how many pending comments will be included, but providing no way to easily *see* those comments, short of closing the text box and opening a second tab to the Conversation view.

**Comment by tonglil** [comment]
+1

Similar request in https://github.com/orgs/community/discussions/35103.

**Comment by robmcgee-isw** [comment]
+1
